CMakeLists.txt

```diff
@@ -0,0 +1 @@
+Index.cmake
```

Comment:
We used to have some issues with Conan when we added CMakeLists.txt. It seems that adding this doesn't cause issues, right?

Reply:
I use it in other projects, but without  conan.

Reply:
We need to test this again.